app/user/rest.py

```diff
@@ -926,3 +926,39 @@ def send_annual_usage_data(user_id, start_year, end_year, markdown_en, markdown_
     send_notification_to_queue(saved_notification, False, queue=QueueNames.NOTIFY)
 
     return jsonify({}), 204
+
+
+def send_requested_report_ready(report):
+    """
+    We are sending a notification to the user to inform them that their requested
+    report is ready.
+    """
+    template = dao_get_template_by_id(current_app.config["REPORT_DOWNLOAD_TEMPLATE_ID"])
+    service = Service.query.get(current_app.config["NOTIFY_SERVICE_ID"])
+    report_service = Service.query.get(report.service_id)
+
+    report_name_en = f"{report.requested_at}-emails-{report_service.name}"
+    report_name_fr = f"{report.requested_at}-courriels-{report_service.name}"
```

Comment:
The time format doesn't match figma and needs to be in the user's time zone:
<img width="647" alt="Screenshot 2025-04-14 at 2 28 21 PM" src="https://github.com/user-attachments/assets/5354bd64-e6a6-467d-b920-19b91a82f27a" />